app/classes/Transvision/ShowResults.php

```diff
@@ -58,6 +58,9 @@ public static function getSuggestionsResults($source_strings, $target_strings, $
         ];
         $flat_output = [];
 
+        // Make sure $max_results is never set to 0, in which case we set it to 10
```

Comment:
I would rephrase this a bit: "Reset $max_results as 10 if it's currently set to 0 (default value when the parameter is not specified in the API request)."